### ISSUE TITLE
508-defect-2 [SCREENREADER] Download (PDF) links need unique alt tags to differentiate between them

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#21017]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Noah

## Details

Links with identical text need an alt attribute with unique text so screen readers can tell them apart

## Acceptance Criteria

- [ ] Add `alt` tag to each link with text

## Proposed Solution (if known)

`<a download="" href="...">Download (PDF)</a>`

should become something like:

`<a download="" href="..." alt="Download Veteran Community Care: General Information PDF">Download (PDF)</a>`

**Comment by brianalloyd** [comment]
Validated, closing ticket in support.
